ui/src/templates/containers/CommunityTemplatesIndex.tsx

```diff
@@ -148,25 +143,14 @@ class UnconnectedTemplatesIndex extends Component<Props> {
     )
   }
 
-  private applyTemplates = async (orgID, templateName) => {
+  private reviewTemplateResources = async (orgID, templateName) => {
     const yamlLocation =
       getRawYamlFromGithub(this.state.currentTemplate) + `/${templateName}.yml`
 
-    const params = {
-      data: {
-        dryRun: true,
-        orgID,
-        remotes: [{url: yamlLocation}],
-      },
-    }
     try {
-      const resp = await postTemplatesApply(params)
-      if (resp.status >= 300) {
-        throw new Error((resp.data as PkgError).message)
-      }
+      const summary = await reviewTemplate(orgID, yamlLocation)
 
-      const summary = (resp.data as TemplateSummary).summary
-      this.props.setActiveCommunityTemplate(summary)
+      this.props.setCommunityTemplateToInstall(summary)
       return summary
     } catch (err) {
       console.error(err)
```

Comment:
Is this the error catch for submitting a valid URL or file location but the API can't return a summary? I am wondering if this needs to be visualized in the UI

Reply:
@alexpaxton i think we'll need to add some ability to display errors to the users. right now the ui is just kind of swallowing them. i'm not quite sure what errors we'll need to show though